package.json

```diff
@@ -26,11 +26,12 @@
   "devDependencies": {
     "@shopify/eslint-plugin": "^44.0.0",
     "@types/better-sqlite3": "^7.6.9",
-    "@types/express": ">=4.17.21",
+    "@types/express": "^4.17.21",
     "@types/fs-extra": "^9.0.13",
     "@types/jest": "^29.5.12",
     "@types/multer": "^1.4.11",
     "@types/node": "^20.11.27",
+    "@types/passport-google-oauth20": "^2.0.16",
```

Comment:
I assume this is here as we only care about using google auth in dev?

Reply:
Thats there because its a typescript "type" definition.  Its not needed for the node app to compile and run.  It just provides the type definitions to the typescript transpiler.